datafusion/substrait/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -0,0 +1,697 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I moved these files and then removed a level of indenting. I did not change any of the tests